src/components/GradesView/index.jsx

```diff
@@ -46,7 +46,7 @@ export class GradesView extends React.Component {
         <SpinnerIcon />
 
         <InterventionsReport />
-        <h4 className="step-message-1">
+        <h4 className="mt-5 font-weight-bold">
```

Comment:
similar question about why we're adding `font-weight-bold` to these heading elements when our Paragon typography styles for headings already has a `font-weight: 700;` applied to make it bold.